site/src/atomics/aspect-ratio.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Given just how useful (very!) these will be for images. Perhaps it's worth a small section in the Image component documentations that mentions aspect-ratio atomics and links to this page?

https://design.learn.microsoft.com/components/image.html